doc/man/bitcoin-cli.1

```diff
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.2.
```

Comment:
not super familiar with this process, but im guessing modifying these files won't cause any issues?